core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -563,6 +564,24 @@ public void updateOobChannelAddresses(ManagedChannel channel, EquivalentAddressG
       throw new UnsupportedOperationException();
     }
 
+    /**
+     * Creates an out-of-band channel for LoadBalancer's own RPC needs, e.g., talking to an external
+     * load-balancer service, that is specified by a target string.  See the documentation on
+     * {@link ManagedChannelBuilder#forTarget} for the format of a target string.
+     *
+     * <p>The target string will be resolved by a {@link NameResolver} created according to the
+     * target string.  The out-of-band channel doesn't have load-balancing. If multiple addresses
+     * are resolved for the target, the "pick_first" policy will be applied.
+     *
+     * <p>The LoadBalancer is responsible for closing unused OOB channels, and closing all OOB
+     * channels within {@link #shutdown}.
+     *
+     * <P>NOT IMPLEMENTED: this method is currently a stub and not yet implemented by gRPC.
+     */
+    public ManagedChannel createResolvingOobChannel(String target) {
```

Comment:
I'm not against a slight different name, but "createResolvingOobChannel" may not be better than "createOobChannel".